src/utils/util.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @method isEmpty
+ * @param {String | Number | Object} value
+ * @returns {Boolean} true & false
+ * @description this value is Empty Check
+ */
+export const isEmpty = (value: string | number | object): boolean => {
+  if (value === null) {
+    return true;
+  } else if (typeof value !== 'number' && value === '') {
+    return true;
+  } else if (typeof value === 'undefined' || value === undefined) {
+    return true;
+  } else if (value !== null && typeof value === 'object' && !Object.keys(value).length) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
this can be optimized, there's no need for having these much if else conditions.

Reply:
Done